pkg/pillar/scripts/device-steps.sh

```diff
@@ -126,16 +126,17 @@ echo
 # /persist/certs/v2tlsbaseroot-certificates.sha256. This is to prepare for a future
 # feature where the controller can update the base file.
 # Note that programatically we add any proxy certificates to the list of roots we trust.
-if [ ! -f /config/v2tlsbaseroot-certificates.pem ]; then
+if [ ! -f /config/v2tlsbaseroot-certificates.pem ] || [ ! -s /config/v2tlsbaseroot-certificates.pem ]; then
```

Comment:
This is redundant @naiming-zededa -- you should simply replace `-f` with `-s`

Reply:
good point. will do.